lib/graphiti_errors/validation/validatable.rb

```diff
@@ -0,0 +1,14 @@
+module GraphitiErrors
+  module Validation
+    module Validatable
+      def render_errors_for(record)
```

Comment:
Is this used anymore? I dont think so and if not lets remove

Reply:
Good call. pushed commit removing it.